src/main/java/com/hrznstudio/titanium/Titanium.java

```diff
@@ -89,6 +98,12 @@ public static void openGui(TileActive tile, ServerPlayerEntity player) {
 
     @Override
     protected void initModules() {
+        ResourceRegistry.scanForReferences();
```

Comment:
I'd move this into an init method in ResourceRegistry rather than throw more in the initModules method.

Reply:
I'd also move the rest of the ResourceRegistry calls from here into the Registry, even if another method, rather than pollute the preInit call